gatsby-browser.js

```diff
@@ -55,4 +55,19 @@ const onClientEntry = () => {
   window.ReactDOM = require('react-dom');
 };
 
-export { wrapPageElement, onInitialClientRender, onClientEntry };
+const shouldUpdateScroll = ({ routerProps: { location } }) => {
+  const transitionDelay = 600;
```

Comment:
Can you put this variable in all caps since its an artificially set constant? like `TRANSITION_DELAY`;

Reply:
fixed 🤠